src/components/Map.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { Marker } from './Marker';
+import CurrentPositionMarker from './CurrentPositionMarker';
+import GoogleMapReact from 'google-map-react';
+import { ClinicService, ClinicType, GeoLocation } from './ClinicFinder';
+
+interface Map {
+  defaultProps: {
+    center: {
+      lat: number;
+      lng: number;
+    };
+    zoom: number;
+  };
+  showCurrentPosition: boolean;
+  customPosition: GeoLocation;
+  lat: number;
+  lng: number;
+  activeClinicSiteId: number;
+  hoveredMarker: number;
+  clinic: string;
+  showSearchMarker: boolean;
+  userGeoLocation: GeoLocation;
+  clinicServices: ClinicService[];
+  toggleInfoCard: (value: number) => void;
+}
+
+const Map = (props: Map) => {
+  const defaultProps = props.defaultProps;
+  const positioning = props.showCurrentPosition;
+  const customPosition = props.customPosition;
+  const userGeoLocation = props.userGeoLocation;
+  const clinicServices = props.clinicServices;
+
+  return (
+    <GoogleMapReact
+      bootstrapURLKeys={{ key: 'AIzaSyCy22mfVK_HzEe6aYr-aV0YE-10qAcWSXQ' }}
```

Comment:
Should this live in the source file? I thought it would belong in a env file or something.

Reply:
doesn't matter bcs bundled in script file